notebooks/pregnancy/maternal_brain_project/r2_pvalues_all_enigma_meshes_new.ipynb

```diff
@@ -0,0 +1,1445 @@
+{
```

Comment:
<p>Nice! Ok, so for this dataset, the first 4 PCs are only 63% of the variance: prob not enough, which can also explain why vertexwise-r2s are not that great.</p><p> </p><p>why having "vertex" twice in: vertex-vertexwise_rec_error?</p><p> </p><p>are these in mm?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geometric-intelligence/herbrain/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>